ludwig/schema/decoders/sequence_decoders.py

```diff
@@ -0,0 +1,92 @@
+from marshmallow_dataclass import dataclass
+
+from ludwig.schema import utils as schema_utils
+from ludwig.schema.decoders.base import BaseDecoderConfig
+
+
+@dataclass
+class SequenceGeneratorDecoderConfig(BaseDecoderConfig):
+
+    type: str = schema_utils.StringOptions(
```

Comment:
One more nit, set `allow_none` to `False` for these fields

Reply:
Ah true, will do that now.